Infra VMs: fix some playbook host patterns

priteau opened the pull request:
The infra-vms group was not added to some playbook host patterns where it should have been. This change fixes that.

TrivialFix

Change-Id: I5df2918035df7577627fd2bd68417beddbcbf848 (cherry picked from commit 5e9affb62c40a885f4d64b0dbcc4bf6576b8779c)

markgoddard commented on ansible/roles/docker/tasks/config.yml:
```diff
@@ -6,6 +6,13 @@
 
+- name: Ensure the path for CA file for private registry exists
+  file:
+    path: "/etc/docker/certs.d/{{ docker_registry }}"
```
Does it need to have any particular mode and/or ownership?

ktibi replied:
no works with become: true

markgoddard commented on ansible/roles/docker/tasks/config.yml:
```diff
@@ -6,6 +6,13 @@
+    path: "/etc/docker/certs.d/{{ docker_registry }}"
+    state: directory
+  become: True
+  when: kolla_docker_registry is not none and docker_registry_ca is not none
```
Should be docker_registry.

markgoddard committed:
Infra VMs: fix some playbook host patterns

The infra-vms group was not added to some playbook host patterns where
it should have been. This change fixes that.

TrivialFix

Change-Id: I5df2918035df7577627fd2bd68417beddbcbf848
(cherry picked from commit 5e9affb62c40a885f4d64b0dbcc4bf6576b8779c)
